test/unit/calculators/state_pension_age_calculator_test.rb

```diff
@@ -60,6 +60,17 @@ class StatePensionAgeCalculatorTest < ActiveSupport::TestCase
           assert_equal @calculator.state_pension_age, '60 years'
         end
       end
+
+      context 'given a state_pension_date on the 29th of Feb when the pension age is 68' do
+        setup do
+          @calculator.stubs(:dob).returns(Date.parse("29 Feb 1980"))
+          @calculator.stubs(:state_pension_date).returns(Date.parse('29 Feb 2048'))
```

Comment:
It seems very odd to me that in the test above this one, we return a date of 1 March 2040 (also a leap year), and it _also_ claims that state pension is exactly 60 years away. That's not right, is it?

Reply:
I completely agree, anyone born in 1980 will not be eligible for their state pension at 60 years old